demosplan/DemosPlanCoreBundle/Application/ConsoleApplication.php

```diff
@@ -41,8 +42,9 @@ public function doRun(
     private function addProjectFolderConsoleDeprecationNotice(OutputInterface $output): void
     {
         if ($this->isDeprecatedFrontController) {
-            $activeProject = $this->getKernel()->getActiveProject();
-
+            $kernel = $this->getKernel();
+            /** @var DemosPlanKernel $kernel */
```

Comment:
In general I would set this comment at the place where the variable is defined (one line above) but no blocker